app/src/main/java/org/sagebionetworks/bridge/researchstack/MPowerOnboardingManager.java

```diff
@@ -40,17 +40,6 @@ public void registerSurveyItemAdapter(GsonBuilder builder) {
         builder.registerTypeAdapter(SurveyItem.class, new CustomSurveyItemAdapter());
     }
 
-    /**
-     * @param item CustomSurveyItem, which will be the type returns from our CustomSurveyItemAdapter
-     * @param factory either a SurveyFactory, or ConsentDocumentFactory subclass
-     * @return a CustomStep object, which can be anything we want it to be
-     */
-    @Override
-    public Step createCustomStep(Context context, SurveyItem item, SurveyFactory factory) {
```

Comment:
I kept this in as an example of how you parse custom steps, which I thought would be valuable for future developers.  Can you add it back in?